README.md

```diff
@@ -0,0 +1,39 @@
+Setting up the environment variables
+------------------------------------
+To setup the Girder instance two users are created: ```esimmonadmin``` and ```esimmon```. The passwords for these users must be set in the ```.env``` file located in ```eSimMon/devops/docker``` under the ```ADMIN_PASSWORD``` and ```ESIMMON_PASSWORD``` keys. Optionally, if a Girder instance has already been built the ```GIRDER_API_KEY``` and ```GIRDER_FOLDER_ID``` can be set in the ```watch.env``` file located in the same directory. If the keys are not set, they will be created automatically when the Girder instance is created.
+
+
+Create and setup the Girder instance
+------------------------------------
+To create the Girder instance, you will need to have set the ```ADMIN_PASSWORD``` and ```ESIMMON_PASSWORD``` keys mentioned above, then run the following command:
+
+```docker-compose -p esimmon -f docker-compose.girder.yml -f docker-compose.ansible.yml up```
```

Comment:
We need to mention the directory we should cd into ...

Reply:
Updated